DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: precommit
 Title: Pre-Commit Hooks
-Version: 0.3.2.9022
+Version: 0.3.2.9023
```

Comment:
this will be gone by reverting to https://github.com/lorenzwalthert/precommit/commit/27f5bdee5a8cb4d37593d077a5960dbfc94ad063.